src/components/LandingSection/LandingSection.js

```diff
@@ -21,7 +22,7 @@ const LandingHeader = styled.h1`
 const ChangeLocaleWrapper = styled.div`
   position: fixed;
   top: ${spacing.xlSize};
-  left: -${spacing.xxxxlSize};
+  left: -100px;
```

Comment:
maybe by next PR, I suggest to switch to rem. For example, you can use this `polished` package. You should be happy with it :P